app/client/build_runner.py

```diff
@@ -146,8 +146,8 @@ def _download_and_extract_results(self, timeout=None):
         """
         timeout_time = time.time() + timeout if timeout else sys.maxsize
 
-        download_artifacts_url = self._master_api.url('build', self._build_id, 'result')
-        download_filepath = 'build_results/artifacts.tar.gz'
+        download_artifacts_url = self._master_api.url('build', self._build_id, 'artifacts.zip')
```

Comment:
Hmm maybe, although it's just an API route so by that argument 'build' should probably also be a constant. I could see arguments for making those constants but kind of want to keep that out of this change.